frontend/src/components/RoutePlanner/RoutePlanner.module.css

```diff
@@ -0,0 +1,93 @@
+ion-modal#favorite_dialogue {
```

Comment:
Why is this styling specific for the favourite dialogue? Wasn't the goal to style the components in a reusable way? And why do the modals require height configuration at all? Shouldn't the height be determined automatically by the size of the content?

Reply:
Was adoped from existing route planner CSS. Will rework to be reusable as a class